utils/generator/README.md

```diff
@@ -0,0 +1,2256 @@
+# Proxy Library RPC Generator
+
+## Overview
```

Comment:
The `README.md` seems to be a bit outdated now. Can we update it to reflect the latest changes like mapping removal and enum simplifications?

Reply:
@vladmu could you have look

Reply:
Readme updated